apps/frontend/src/modules/users/users-api.ts

```diff
@@ -29,6 +33,23 @@ class UserApi extends BaseHTTPApi {
 
 		return await response.json<UserGetAllResponseDto>();
 	}
+
+	public async update(
```

Comment:
we will use update for put
```suggestion
	public async patch(
```